### PR TITLE
Fix AWS ELB max 20 issue

### PR DESCRIPTION
**Pull request by loganhz**
[issue rancher/rancher#7307](https://github.com/rancher/rancher/issues/7307)

### CODE REVIEW

**Comment by loganhz** [comment]
@yasker fixed. Need a QA to test it.
thanks a lot

**Comment by brooksgarrett** [comment]
Any chance of this getting merged?

**Comment by yasker** [comment]
@brooksgarrett We're waiting for QA to test. In the meantime, you can also try it and let us know. @loganhz has instructions at https://github.com/rancher/rancher/issues/7307

**Comment by brooksgarrett** [comment]
@yasker Thanks for the update. I pulled the patch submitted by @loganhz this morning and built a container for our private repo. It resolved the issue immediately and I have not yet noticed any errors or issues. I'll post back if anything comes up but so far it is looking good on my end!

Thanks for all you guys (and girls) do!

**Comment by yasker** [comment]
@brooksgarrett Glad it helps! Feel free to let us know if any issue pops up. As I said, we will merge it after it passed our QA test, which should happen soon.

File: providers/elbv1/elbv1svc/elb_classic.go [hunk 1]
```diff
@@ -360,3 +352,35 @@ func (svc *ELBClassicService) GetRegisteredInstances(loadBalancerName string) ([
...
+		awsNames := make([]*string, len(names))
+		for i, n := range names {
+			awsNames[i] = aws.String(n)
+		}
+
+		params := &elb.DescribeTagsInput{
+			LoadBalancerNames: awsNames,
+		}
+		resp, err := svc.elbc.DescribeTags(params)
+		tagDescriptions = append(tagDescriptions, resp.TagDescriptions...)
```
Comment (yasker) [inline providers/elbv1/elbv1svc/elb_classic.go]: Keep the `err` statement following the assignment, otherwise you're risking a NPE.

**Review by yasker — CHANGES_REQUESTED** [review]

**Comment by loganhz** [comment]
@yasker done

**Review by yasker — APPROVED** [review]

**Comment by yasker** [comment]
LGTM.

**Comment by alena1108** [comment]
LGTM

**Pull request merged**